MotorizationRate.ipynb

```diff
@@ -148,7 +148,7 @@
     }
```

Comment:
<p>It can be useful to define</p><p> </p><pre class="ql-syntax" spellcheck="false">target = "Taux de motorisation Commune"
</pre><p> </p><p>and then use <code>taget</code>  instead in the rest of the code.</p><p> </p><ul><li>It explicits the target in the code</li><li>It explicits when we are talking about the target in the following code</li><li>It makes the following code more independant of the input data (thus it becomes reusable, or can adapt more easily)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Adanel/MotorizationRate/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>